exercises/practice/forth/src/test/java/ForthEvaluatorTest.java

```diff
@@ -146,42 +111,25 @@ public void testThatIntegerDivisionIsUsed() {
     @Ignore("Remove to run test")
     @Test
     public void testErrorIfDividingByZero() {
-        IllegalArgumentException expected =
-            assertThrows(
-                IllegalArgumentException.class,
-                () -> forthEvaluator
-                    .evaluateProgram(Collections.singletonList("4 0 /")));
-
-        assertThat(expected)
-            .hasMessage("Division by 0 is not allowed");
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> forthEvaluator.evaluateProgram(Collections.singletonList("4 0 /")))
+                .withMessage("Division by 0 is not allowed");
     }
 
     @Ignore("Remove to run test")
     @Test
     public void testErrorIfDivisionAttemptedWithNothingOnTheStack() {
-        IllegalArgumentException expected =
-            assertThrows(
-                IllegalArgumentException.class,
-                () -> forthEvaluator
-                    .evaluateProgram(Collections.singletonList("/")));
-
-        assertThat(expected)
-            .hasMessage(
-                "Division requires that the stack contain at least 2 values");
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> forthEvaluator.evaluateProgram(Collections.singletonList("/")))
+                .withMessage("Division requires that the stack contain at least 2 values");
     }
 
-    @Ignore("Remove to run test")
+    //@Ignore("Remove to run test")
```

Comment:
Please revert this change, we want all tests except the first test to be ignored by default.